client/src/main/scala/com.gu.contentapi.client/Parameters.scala

```diff
@@ -25,7 +25,7 @@ trait Parameters[Owner <: Parameters[Owner]] { self: Owner =>
 
   def withParameters(parameterMap: Map[String, Parameter]): Owner
 
-  protected def parameterHolder: Map[String, Parameter]
+  def parameterHolder: Map[String, Parameter]
```

Comment:
This is a breaking change for anyone that defines a query. Is it really necessary?

Reply:
I don't like it either, as it surfaces the underlying implementation. Will change that.